pkg/jobs/registry.go

```diff
@@ -298,15 +298,17 @@ func (r *Registry) Run(ctx context.Context, ex sqlutil.InternalExecutor, jobs []
 	}
 	log.Infof(ctx, "scheduled jobs %+v", jobs)
 	buf := bytes.Buffer{}
+	usingSQLLiveness := r.startUsingSQLLivenessAdoption(ctx)
 	for i, id := range jobs {
-		select {
-		case r.adoptionCh <- struct{}{}:
-		case <-r.stopper.ShouldQuiesce():
-			return stop.ErrUnavailable
-		case <-ctx.Done():
-			return ctx.Err()
+		if !usingSQLLiveness || i == 0 {
```

Comment:
please add a comment explaining the if condition

Reply:
Done.